apps/web-mzima-client/src/app/map/map.component.ts

```diff
@@ -77,58 +77,62 @@ export class MapComponent extends MainViewComponent implements OnInit {
       sessionService,
       breakpointService,
     );
-    this.filtersSubscription$ = this.postsService.postsFilters$.pipe(takeUntilDestroy$());
+    if (this.sessionService.hasSiteConfiguration()) {
+      this.filtersSubscription$ = this.postsService.postsFilters$.pipe(takeUntilDestroy$());
+    }
   }
 
   ngOnInit() {
-    this.reInitParams();
-    this.route.params.subscribe(() => {
-      this.initCollection();
-    });
-    this.initFilterListener();
-    this.mapConfig = this.sessionService.getMapConfigurations();
-
-    const currentLayer =
-      mapHelper.getMapLayers().baselayers[this.mapConfig.default_view!.baselayer];
-
-    this.leafletOptions = {
-      minZoom: 1,
-      maxZoom: 22,
-      scrollWheelZoom: true,
-      zoomControl: false,
-      layers: [tileLayer(currentLayer.url, currentLayer.layerOptions)],
-      center: [this.mapConfig.default_view!.lat, this.mapConfig.default_view!.lon],
-      zoom: this.mapConfig.default_view!.zoom,
-    };
-    this.markerClusterOptions.maxClusterRadius = this.mapConfig.cluster_radius;
-
-    this.mapReady = true;
-
-    this.eventBusService.on(EventType.SearchOptionSelected).subscribe({
-      next: (option) => {
-        this.map.setZoom(12);
-        this.map.panTo({ lat: option.lat, lng: option.lon });
-      },
-    });
+    if (this.sessionService.hasSiteConfiguration()) {
+      this.reInitParams();
+      this.route.params.subscribe(() => {
+        this.initCollection();
+      });
+      this.initFilterListener();
+      this.mapConfig = this.sessionService.getMapConfigurations();
+
+      const currentLayer =
+        mapHelper.getMapLayers().baselayers[this.mapConfig.default_view!.baselayer];
+
```

Comment:
I see some things about map around this area of the code... And how it looks on the UI.

Wouldn't it just be better to approach this like it's done in private deployments?
We usually have no business with the map or any other thing going on, on the UI.
Plus the background behind the modal will be cleaner.

#
#### screenshot with private deployment example
<img width="1406" alt="Screenshot 2024-08-06 at 04 47 57" src="https://github.com/user-attachments/assets/d29f6adc-5b1e-4d75-9bbf-80ac1b0453ec">

#

> Again, probably a product and/or design related concern...

> But do let me know if I'm over thinking the it, or this is beyond the scope of this PR.